src/test/ui/methods/issues/issue-61525.stderr

```diff
@@ -0,0 +1,39 @@
+error[E0277]: the size for values of type `dyn ToString` cannot be known at compilation time
```

Comment:
This error is actually more verbose than what it used to (erroneously) say before, but it's more consistent with an error you'd get if you, for example, passed any other DST as a generic arg. It also makes the second error, the mismatched types (`""` against `dyn ToString`) not be suppressed any longer, which IMO is better.